pepper-apis/docs/specification/src/endpoints/user.profile.yml

```diff
@@ -16,7 +16,7 @@ get:
   responses:
     200:
       $ref: '../pepper.yml#/components/responses/ProfileResponse'
-    400:
+    404:
```

Comment:
@fat23cat 
Pavel: return code in response is changed from 400 to 404. Can you please check if this needs any front-end change incase front-end is checking for 400.

Reply:
Yep, we need to change code on the frontend,I will open PR.